kolibri/locale/zh_Hans/LC_MESSAGES/kolibri.plugins.coach.side_nav-messages.json

```diff
@@ -0,0 +1,127 @@
+{
```

Comment:
@marcellamaki - we probably want to change how we're handling this in the side-nav messages, this will load all these messages into every page.